Random

folkertvanheusden opened the pull request:
Crypto random may not be needed, but rand() is not random at all.
    Also coverity complains about it :-)

davidmoreno commented:
I like the idea, but the implementation opens the urandom on every call, which is a bit overkill. I reworked it to use C++11 random number generators and added license and namespace.

folkertvanheusden commented:
Thanks!
(and the other merges as well)

On Mon, Aug 2, 2021 at 8:48 PM David Moreno Montero <
***@***.***> wrote:

> I like the idea, but the implementation opens the urandom on every call,
> which is a bit overkill. I reworked it to use C++11 random number
> generators and added license and namespace.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/davidmoreno/rtpmidid/pull/58#issuecomment-891251024>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUN5IW4QHLKMGYAAMNTFKUTT23R55ANCNFSM5BLPV3AQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
